src/main/java/org/powerflows/dmn/engine/evaluator/entry/expression/provider/GroovyExpressionEvaluationProvider.java

```diff
@@ -17,28 +17,62 @@
 package org.powerflows.dmn.engine.evaluator.entry.expression.provider;
 
 import org.powerflows.dmn.engine.evaluator.context.ModifiableContextVariables;
+import org.powerflows.dmn.engine.evaluator.entry.expression.provider.script.ScriptEngineProvider;
+import org.powerflows.dmn.engine.evaluator.entry.expression.provider.script.bindings.ContextVariablesBindings;
+import org.powerflows.dmn.engine.evaluator.exception.EvaluationException;
+import org.powerflows.dmn.engine.model.decision.expression.Expression;
 import org.powerflows.dmn.engine.model.decision.field.Input;
 import org.powerflows.dmn.engine.model.decision.rule.entry.InputEntry;
 import org.powerflows.dmn.engine.model.decision.rule.entry.OutputEntry;
 import org.powerflows.dmn.engine.model.evaluation.result.EntryResult;
 
+import javax.script.Bindings;
+import javax.script.ScriptEngine;
+import javax.script.ScriptException;
+
 /**
  * This provider should be moved to external jar as an optional dependency
  */
 class GroovyExpressionEvaluationProvider extends AbstractExpressionEvaluationProvider {
```

Comment:
This class likely will be refactored in the next phase of development since looks like this code can be re-used for more scripts evaluation. I.e. JavaScript.